tests/test-cases.js

```diff
@@ -2018,6 +2027,51 @@ describe('Unparse Tests', function() {
 
 
 var CUSTOM_TESTS = [
+	{
+		description: "Pause and resume works with headers and duplicate fields (Regression Test for Bug #985)",
+		expected: [[
+			["Column 1", "Column 2", "Column 3", "Column 4", "Column 5"],
```

Comment:
Do we really need to have 5 rows? Can we make the test simpler with less rows?

Reply:
Probably not. I reduced it to two rows and four columns, enough to make sure we could test column value duplicates and that the headers didn't shift between pause/resume stages.